tests/mock_tables/config_db.json

```diff
@@ -831,8 +831,7 @@
         "mac": "1d:34:db:16:a6:00",
         "platform": "x86_64-mlnx_msn3800-r0",
         "peer_switch": "sonic-switch",
-        "type": "ToRRouter",
-        "subtype": "DualToR"
```

Comment:
Why remove this? We need to test for DualToR, right?

Reply:
I updated the test so that we can mock this dynamically per test case, it's configured for the dual ToR cases individually: https://github.com/theasianpianist/sonic-utilities/blob/de5c0332defd6585f9096e0970ef145bc706204c/tests/route_check_test_data.py#L288
https://github.com/theasianpianist/sonic-utilities/blob/de5c0332defd6585f9096e0970ef145bc706204c/tests/route_check_test_data.py#L336